From 86b86403e1862bd2e170acc035b5046c0764281c Mon Sep 17 00:00:00 2001
Message-ID: <86b86403e1862bd2e170acc035b5046c0764281c.1692870627.git.jdenemar@redhat.com>
From: Michal Privoznik <mprivozn@redhat.com>
Date: Thu, 17 Aug 2023 17:43:54 +0200
Subject: [PATCH] qemu_domain: Drop unused variables from
 qemuDomainChrDefDropDefaultPath()

In mu previous commits I've moved internals of
qemuDomainChrDefDropDefaultPath() into a separate function
(qemuDomainChrMatchDefaultPath()) but forgot to remove @buf and
@regexp variables which are now unused.

Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
(cherry picked from commit b07640bb438d21e592d66ca8367904d82838602f)
Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2233744
Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
---
 src/qemu/qemu_domain.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/qemu/qemu_domain.c b/src/qemu/qemu_domain.c
index 97e534a1f0..8fbcb68265 100644
--- a/src/qemu/qemu_domain.c
+++ b/src/qemu/qemu_domain.c
@@ -5204,8 +5204,6 @@ qemuDomainChrDefDropDefaultPath(virDomainChrDef *chr,
                                 virQEMUDriver *driver)
 {
     g_autoptr(virQEMUDriverConfig) cfg = NULL;
-    g_auto(virBuffer) buf = VIR_BUFFER_INITIALIZER;
-    g_autofree char *regexp = NULL;
 
     if (chr->deviceType != VIR_DOMAIN_CHR_DEVICE_TYPE_CHANNEL ||
         chr->targetType != VIR_DOMAIN_CHR_CHANNEL_TARGET_TYPE_VIRTIO ||
-- 
2.42.0
